kpi/serializers/v2/service_usage.py

```diff
@@ -12,21 +12,37 @@ class AssetUsageSerializer(serializers.HyperlinkedModelSerializer):
         view_name='asset-detail',
     )
     asset__name = serializers.ReadOnlyField(source='name')
+    nlp_usage_current_month = serializers.SerializerMethodField()
+    nlp_usage_all_time = serializers.SerializerMethodField()
+    storage_bytes = serializers.SerializerMethodField()
     submission_count_current_month = serializers.SerializerMethodField()
     submission_count_all_time = serializers.SerializerMethodField()
-    storage_bytes = serializers.SerializerMethodField()
 
     class Meta:
         model = Asset
         lookup_field = 'uid'
         fields = (
             'asset',
             'asset__name',
+            'nlp_usage_current_month',
+            'nlp_usage_all_time',
+            'storage_bytes',
             'submission_count_current_month',
             'submission_count_all_time',
-            'storage_bytes',
         )
 
+    def get_nlp_usage_current_month(self, asset):
+        if not asset.has_deployment:
+            return 0
+
+        return asset.deployment.current_month_nlp_tracking
+
+    def get_nlp_usage_all_time(self, asset):
+        if not asset.has_deployment:
+            return 0
```

Comment:
`asset.deployment.nlp_tracking` returns a dict when `asset.has_deployment` is True. 
It should return an empty dict. Otherwise, the serializer will crash on [line 126](https://github.com/kobotoolbox/kpi/pull/4216/files#diff-0af4c1e95adb7b8c91cf7b2235be21323b5ab7e0f43d39ef833d4eb8676cda89R126) 


Please add a unit test when asset has no deployments. Just to validate no 500 errors are raised. 🙏